### ISSUE TITLE
Play several videos in RecyclerView at the same time?

### ISSUE THREAD

**Issue by amanzan** [tcking/GiraffePlayer2#63]
Is it possible to play several videos in a RecyclerView at the same time?

**Comment by tcking** [comment]
the player design for playing one video at a time，because the hardware that can only be used by one player

**Comment by amanzan** [comment]
That is not true, of course hardware can play more than one video at the same time :)
I'll try to take a look at the code to see if I can create a PR, but any hint would be appreciated!

**Comment by tcking** [comment]
you can try it
